.packit.yaml

```diff
@@ -77,7 +77,6 @@ jobs:
     targets:
       - fedora-stable
```

Comment:
`fedora-stable` still includes F40 that fails

Reply:
Actually not sure what we want to do here… we could probably omit `targets` altogether and deduce from the Copr settings, but that would mean that we need to remove this entry from all of `specfile, ogr, packit`, cause either of them could “reset” the F40 and EL9 back.

Reply:
I have already started removing things in packit, https://github.com/packit/packit/pull/2549/files. At least we should remove epel9 there. And use `fedora-latest` instead?

Reply:
> if we intend to move our deployments with Fedora releases, then I guess we could use `fedora-latest`

Hm, but if we forget will we be able to notice (soon enough) that the images contain some old builds?

Reply:
As long as we're talking about `packit/packit-stable` Copr, I would say it would be enough to deduce the targets from Copr and adjust the release in images and Copr repo at the same time, manually.

Reply:
That also means we should eradicate `targets: […]` in all repos that build in `packit/packit-stable`…

Reply:
I am not sure I follow, why using `fedora-latest` our images could use old builds?

Reply:
> I am not sure I follow, why using `fedora-latest` our images could use old builds?

If we bump (or not bump) the base image without adjusting the Copr settings (or config here), we will start pulling the dependencies from official Fedora repository (which will not necessarily be the same as us moving the stable and using Copr repo)

Reply:
what about going with `fedora-latest-stable`?